### PR TITLE
Allow rebuilds to invoke the Slack Plugin for successful rebuilds

### LINKED ISSUE #2: Update status of master builds is not announced in IRC
When one of the sub-jobs of a master project is fixed, the status of the master project updates in the Jenkins UI, but no IRC announcement is made.

### PR DESCRIPTION
**Pull request by ivantsepp**
I knew absolutely nothing about jenkins and its architecture so it was interesting to learn about it. I've used these resources:

https://github.com/jenkinsci/jenkins/blob/master/core/src/main/resources/lib/form/optionalBlock.jelly
https://wiki.jenkins-ci.org/display/JENKINS/Jelly+form+controls
https://wiki.jenkins-ci.org/display/JENKINS/Basic+guide+to+Jelly+usage+in+Jenkins
https://wiki.jenkins-ci.org/display/JENKINS/Dependencies+among+plugins
https://wiki.jenkins-ci.org/display/JENKINS/Dependencies+among+plugins#Dependenciesamongplugins-Optionaldependencies
https://wiki.jenkins-ci.org/display/JENKINS/Tips+for+optional+dependencies
https://github.com/jenkinsci/slack-plugin

Fixes #2

### CODE REVIEW

File: src/main/java/com/etsy/jenkins/MasterBuild.java [hunk 1]
```diff
@@ -76,6 +78,14 @@ public int getMaxRetries() {
     this.maxRetries = maxRetries;
   }
 
+  public boolean getNotifyOnRebuild() {
+    return this.notifyOnRebuild;
+  }
+
+  /*package*/ void setNotifyOnRebuild(boolean notifyOnRebuild) {
```
Comment (ivantsepp) [inline src/main/java/com/etsy/jenkins/MasterBuild.java]: What are these `/*package*/` comments and do they have any significance

**Comment by rvml** [comment]
good work! do you have any examples / screenshots of a slack notification?

**Comment by ivantsepp** [comment]
@rvml yup, checkout ivan-bots channel

File: src/main/java/com/etsy/jenkins/MasterBuild.java (hunk 1 above)
Comment (rvml) [inline src/main/java/com/etsy/jenkins/MasterBuild.java]: Seems like a Java construct to add documentation for a package

**Comment by ivantsepp** [comment]
ah gotcha thanks!

**Comment by rvml** [comment]
lgtm! please let us know what test suite you plan on adding this rebuild notification to.

**Pull request merged**